dev-python/neovim-remote/neovim-remote-9999.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-PYTHON_COMPAT=( python3_{7,8} )
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8,9} )
 
```

Comment:
Should have an empty line after `EAPI` instead, but not necessarily before `inherit`.
Tests pass?
What about python3_10?

Reply:
Tests passed, currently working on python3_10.